File: IntegratedDigitalAPI/IntegratedImplementation/DTOS/Training/TrainerDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace IntegratedImplementation.DTOS.Training
{
    public  record TrainerPostDto
    {
        public Guid TrainingId { get; set; }       
        public string FullName { get; set; }
        public string Email { get; set; }
        public string PhoneNumber { get; set; }
        public string CreatedById { get; set; }

    }

    public record TrainerGetDto
    {
        public string FullName { get; set; }
        public string Email { get; set; }
        public string PhoneNumber { get; set; }
        public bool IsEmailSent { get; set; }
    }

    public record TrainerEmailDto
    {
        public string FullName { get; set; }
        public string Email { get; set; }

        public string TrainingId { get; set; }  
    }

 
}

File: client/src/app/model/Training/TrainerDto.ts
export interface ITrainerGetDto{

    id? :string
    phoneNumber:string
    email:string
    fullName:string
}

export interface ITrainerPostDto{
    CreatedById?:string
    PhoneNumber:string
    Email:string
    FullName:string
    TrainingId:string
    Id?:string
}